velox/duckdb/conversion/DuckParser.h

```diff
@@ -86,6 +86,8 @@ struct IExprWindowFunction {
       orderBy;
 };
 
-const IExprWindowFunction parseWindowExpr(const std::string& windowString);
+const IExprWindowFunction parseWindowExpr(
+    const std::string& windowString,
```

Comment:
This is a nice change, but it should have been introduced in a separate PR with a test. The options parameter should not have default value for consistency with other functions.

@zhli1142015 Would you create a new PR to remove default value for 'options' and add a test?

Reply:
Sure, will remove it in a separate pr.

Reply:
@zhli1142015 Thanks.